docker-compose.yml

```diff
@@ -14,6 +14,9 @@ services:
       REMOTE_USER: blalbrit@stanford.edu
       RAILS_LOG_TO_STDOUT: 'true'
       SOLR_URL: http://solr:8983/solr/argo
+      # Allow bulk action logs to be written (can't write to /tmp)
+      SETTINGS__BULK_METADATA__DIRECTORY: '/app/tmp'
+      SETTINGS__BULK_METADATA__TEMPORARY_DIRECTORY: '/app/tmp/tmp'
```

Comment:
okay ... uh ... what?

Reply:
Until I overrode these values, I didn't see bulk actions successfully write their log files, thus rendering bulk actions inoperable when testing locally using containers. This change only affects local dev and testing.